### ISSUE TITLE
Board Tooltip following cursor

### ISSUE THREAD

**Issue by BLOODWOLF333** [MegaMek/megamek#362]
Tooltip behavior is unwanted/odd.  @Ralgith and @arlith are aware, this is just a reminder.  If you hover your cursor over a hex long enough to get the tooltip window to pop up and then move your mouse down to click fire or some other button the tooltip window follows your mouse down to the button panel and stays open for whatever the players delay time is set at and just covers up the map and gets in the way.  Other programs you would expect the tooltip to close on mouse movement or at least once you move the mouse away from the hex once your done with it.
```ralgith [1:34 PM]  
I’ve had it follow my mouse __off__ the MM window
[1:34]  
Which… is very wrong.

bloodwolf [1:35 PM]  
i thought so
[1:35]  
my thinking says it should close the tooltip as soon as my mouse moves away from the hex, im done with it so close

ralgith [1:35 PM]  
It should close as soon as there is any mouse movement… like… oh… every other program?```

**Comment by Dylan-M** [comment]
Please add the details anyway, we forget things.

**Comment by BLOODWOLF333** [comment]
So did you fix this @Qwertronix ?

**Comment by Qwertronix** [comment]
This one I did: https://github.com/MegaMek/megamek/pull/772 

So that should be a fix for the part where it stays stuck open when going down to the buttons, but I didn't touch the code that keeps moving and refreshing the tooltip when moving across the board.

**Comment by arlith** [comment]
Fix for this in PR #795.

**Comment by NickAragua** [comment]
I believe we can call this fixed.
